README.md

```diff
@@ -12,17 +12,25 @@ tsschecker is a powerful tool to check TSS signing status on combinations of var
 tsschecker is not only meant to be used to check firmware signing status, but also to explore Apple's TSS servers.<br/>
 By using all of its customization possibilities, you might discover a combination of devices and firmware versions that is getting signed but wasn't getting signed before. 
 
-# About nonces:
-## recommended generators for saving tickets:
-* `0xbd34a880be0b53f3` // default on the Electra, Chimera and Odyssey jailbreak apps.
-* `0x1111111111111111` // default on the unc0ver jailbreak app.
+# About Nonces:
+A [Nonce](https://wikipedia.org/wiki/Cryptographic_nonce) ("Number-used-ONCE") is a randomly generated value that is used to randomize apple's signed hash blobs.
 
-## Nonce Entangling (Apple A12/S4 and newer)
-Newer devices, such as the iPhone XR or the Apple Watch Series 4 (and any device newer) have nonce-entangling.
+it is created by the device with a nonce seed (generator) and then hashes that seed to create the nonce.<br/>On arm64e devices the nonce is also encrypted with the device's UID Key, see "Nonce Entangling" for more details.
 
-this means any boot nonce generated by your device is now also UID derived, and consequently device-specific.<br/>to save usable tickets for a newer device, you need to get the boot nonce that your device actually generates from your generator.
+## Recommended nonce-seeds (Generators) for saving tickets:
+* `0xbd34a880be0b53f3` // default on the [Electra](https://coolstar.org/electra/), [Chimera](https://chimera.coolstar.org/) and [Odyssey](https://theodyssey.dev/) jailbreak apps.
+* `0x1111111111111111` // default on the [unc0ver](https://unc0ver.dev) jailbreak app.
 
-for information on how to get your actual boot nonce, [see this post on r/jailbreak](https://www.reddit.com/r/jailbreak/comments/cssh8f/tutorial_easiest_way_to_save_blobs_on_a12/).
+## Nonce Entangling (arm64e devices)
+arm64e devices such as the iPhone XR, Apple Watch Series 4 and all newer devices have nonce-entangling.
+
+Nonce Entangling works by further randomizing the boot nonce by encrypting it with the device's [unique ID key](https://www.theiphonewiki.com/wiki/UID_key),<br/>
+making the nonce created from the generator specific to that device only.
```

Comment:
Not exactly. Yes UID is involved, however, there is a constant key encrypted by the UID, that encrypted key then is used to encrypt the 16 digit boot nonce. Then its hashed to become apnonce.

Reply:
duplicate of the conversation below.